### PR TITLE
Made the width of the Social media share buttons equal

### LINKED ISSUE #1334: All the social media share buttons should be of equal width
**Actual Behaviour**

Currently, the social media buttons are not of equal width.

**Expected Behaviour**

They should be of equal width.

**Screenshots of the issue**
screenshot from 2018-06-14 12-51-54

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Akshat-Jain**
Fixes #1334 

**Changes:** Now the social media share buttons are of equal width.

**Demo Link:** https://pr-1335-fossasia-susi-web-chat.surge.sh

**Screenshots for the change:** 

**Before:**
screenshot from 2018-06-14 12-51-54

**After:**
screenshot from 2018-06-14 13-15-32

### CODE REVIEW

**Comment by AnupKumarPanwar** [comment]
LGTM but try to remove _Mixed spaces and tabs_  kind of issues in Codacy

File: src/components/ChatApp/Settings/ShareOnSocialMedia.js [hunk 1]
```diff
@@ -17,74 +22,71 @@ export default class ShareOnSocialMedia extends Component{
...
             window.FB.ui({
             method: 'share',
             href: 'https://chat.susi.ai',
             caption: 'SUSI by FOSSASIA',
-						hashtag:'#FOSSASIA',
-						quote:'Lets chat with susi, the open source personal assistant'
-            }, function(response){});
-          }
-				}
+			hashtag:'#FOSSASIA',
```
Comment (PrP-11) [inline src/components/ChatApp/Settings/ShareOnSocialMedia.js]: Please check indentation.

File: src/components/ChatApp/Settings/ShareOnSocialMedia.js [hunk 2]
```diff
@@ -102,9 +105,9 @@ export default class ShareOnSocialMedia extends Component{
 								<RaisedButton
                 					label={<Translate text="Share on Facebook" />}
                 					style={this.state.style}
-													backgroundColor='#3B5998'
+									backgroundColor='#3B5998'
                 					labelColor='#fff'
```
Comment (PrP-11) [inline src/components/ChatApp/Settings/ShareOnSocialMedia.js]: same

File: src/components/ChatApp/Settings/ShareOnSocialMedia.js [hunk 3]
```diff
@@ -102,9 +105,9 @@ export default class ShareOnSocialMedia extends Component{
...
-													icon={<FontIcon className="fa fa-facebook" />}
+									icon={<FontIcon className="fa fa-facebook" />}
                 					keyboardFocused={false}
```
Comment (PrP-11) [inline src/components/ChatApp/Settings/ShareOnSocialMedia.js]: Same

File: src/components/ChatApp/Settings/ShareOnSocialMedia.js [hunk 4]
```diff
@@ -115,9 +118,9 @@ export default class ShareOnSocialMedia extends Component{
 								<RaisedButton
                 					label={<Translate text="Share on Twitter" />}
                 					style={this.state.style}
-													backgroundColor='#00aced'
-													labelColor='#fff'
-													icon={<FontIcon className="fa fa-twitter" />}
+									backgroundColor='#00aced'
+									labelColor='#fff'
+									icon={<FontIcon className="fa fa-twitter twitterIcon" />}
                 					keyboardFocused={false}
```
Comment (PrP-11) [inline src/components/ChatApp/Settings/ShareOnSocialMedia.js]: same

File: src/components/ChatApp/Settings/ShareOnSocialMedia.js [hunk 5]
```diff
@@ -128,9 +131,9 @@ export default class ShareOnSocialMedia extends Component{
 								<RaisedButton
                 					label={<Translate text="Share on Google+" />}
                 					style={this.state.style}
-													backgroundColor='#d34836'
+									backgroundColor='#d34836'
                 					labelColor='#fff'
```
Comment (PrP-11) [inline src/components/ChatApp/Settings/ShareOnSocialMedia.js]: same

**Comment by Akshat-Jain** [comment]
@AnupKumarPanwar @PrP-11 Made the changes. Please review. Thanks :-)

**Comment by mariobehling** [comment]
This is actually low priority. The real feature here should be "connect accounts" instead of "share accounts". Therefore taking care of such things here is adding little value. We need more real features solved. Please help with that.

**Pull request merged**